### DISCUSSION TITLE
Welcome to UsTaxes Discussions!

### DISCUSSION THREAD

**Discussion by zakpatterson** [ustaxes/UsTaxes#147] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by James3412** [comment]
I'm looking for a good free tax program that easy to use. It this the program That want
